### ISSUE TITLE
Issue with host-based locale and non-native routes

### ISSUE THREAD

**Issue by tagliala** [enriclluelles/route_translator#91]
Hi,

I noticed that when using the host-based locale option, you are also able to get all the localized routes.

This is a failing test:

``` rb
  def test_non_native_path
    ## es route on ru com
    host! 'ru.testapp.com'
    get URI.escape('/dummy')
    assert_response :not_found
  end
```

```
$ bundle exec ruby -I"lib:test" test/integration/host_locales_test.rb -n /test_non_native_path/
Run options: -n /test_non_native_path/ --seed 21505

# Running:

F

Finished in 0.027698s, 36.1037 runs/s, 36.1037 assertions/s.

  1) Failure:
HostLocalesTest#test_non_native_path [test/integration/host_locales_test.rb:63]:
Expected response to be a <not_found>, but was <200>.
Expected: 404
  Actual: 200

1 runs, 1 assertions, 1 failures, 0 errors, 0 skips
```

I must dig into this, but if you already know what it's going on please help me

/cc @gi-lunaweb

**Comment by enriclluelles** [comment]
hey man! real sorry here. New work going on and real busy, will try to take a look today!

**Comment by tagliala** [comment]
@enriclluelles don't worry, I'm also very busy so I understand :disappointed:

**Comment by tagliala** [comment]
(ps: sorry for the ping)

**Comment by tagliala** [comment]
https://github.com/enriclluelles/route_translator/tree/fix-native-routes-in-host-locale <-- branch with a failing test. Help would be really appreciated

**Comment by rogercampos** [comment]
I just saw this issue, I think this solves it https://github.com/enriclluelles/route_translator/pull/167

**Comment by tagliala** [comment]
Fixed in the just released 6.0.0.alpha1
